### PR TITLE
tags linked to the page listing book having same tag

### LINKED ISSUE #300: Select all books with a given tag
Currently it is not possible in the bookshelf view to 
- get the number of books which have a given tag,
- select a tag and get the list of related books. 

The database must offer this service and each tag in the bookshelf must act as a button, leading to a search result page for this tag. The breadcrumb at the top of the page must reflect this search by tag.

### PR DESCRIPTION
**Pull request by manuellagrand**
This PR contains tags link to the page listing books having the same tag and it hide number of tag.

fix #300

### CODE REVIEW

File: src/renderer/components/catalog/GridTagButton.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        this.props.findByTag({ tag: this.props.name});
+        console.log("books: ", this.props.tag.length);
+    }*/
+
+    public render(): React.ReactElement<{}> {
+        return (
+            <Link
+            to={{pathname: `/library/search/tag/${this.props.name}`}}>
+                {this.props.name}
+                {/*<div id={style.count}>
```
Comment (clebeaupin) [inline src/renderer/components/catalog/GridTagButton.tsx]: Dot not commit useless comments

File: src/renderer/components/catalog/GridTagButton.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+export class GridTagButton extends React.Component<ButtonTagProps> {
+
+    public constructor(props: any) {
+        super(props);
+
+    }
+
+    /*public componentDidMount() {
```
Comment (clebeaupin) [inline src/renderer/components/catalog/GridTagButton.tsx]: Dot not commit useless comments

Comment (manuellagrand) [reply src/renderer/components/catalog/GridTagButton.tsx]: it isn't useless. it's a feature on standby that will allow me to show the number of tag

Comment (clebeaupin) [reply src/renderer/components/catalog/GridTagButton.tsx]: ok

**Pull request merged**